examples/todo/src/controllers/ping.controller.ts

```diff
@@ -0,0 +1,52 @@
+import {Request, RestBindings, get, ResponseObject} from '@loopback/rest';
+import {inject} from '@loopback/core';
+
+/**
+ * OpenAPI response for ping()
+ */
+const PING_RESPONSE: ResponseObject = {
+  description: 'Ping Response',
+  content: {
+    'application/json': {
+      schema: {
+        type: 'object',
+        title: 'PingResponse',
+        properties: {
+          greeting: {type: 'string'},
+          date: {type: 'string'},
+          url: {type: 'string'},
+          headers: {
+            type: 'object',
+            properties: {
+              'Content-Type': {type: 'string'},
+            },
+            additionalProperties: true,
+          },
+        },
+      },
+    },
+  },
+};
+
+/**
+ * A simple controller to bounce back http requests
+ */
+export class PingController {
+  constructor(@inject(RestBindings.Http.REQUEST) private req: Request) {}
+
+  // Map to `GET /ping`
+  @get('/ping', {
+    responses: {
+      '200': PING_RESPONSE,
+    },
+  })
+  ping(): object {
+    // Reply with a greeting, the current time, the url, and request headers
+    return {
+      greeting: 'Hello from LoopBack',
+      date: new Date(),
+      url: this.req.url,
+      headers: Object.assign({}, this.req.headers),
+    };
+  }
+}
```

Comment:
Let's remove `ping.controller.ts`. There is no need to have it for examples.

Reply:
I wondered about that when I compared what I created by hand vs what is in the finished sample.
Why do we delete it?

Reply:
I propose it to be deleted so that the example application stays focused without unrelated code.

Reply:
I see. ok. It is part of what CLI generates. But I will delete it.